### PR TITLE
Patch posting of trigger activation.

### LINKED ISSUE #2611: trigger posting post akka port generates errors
inspecting the controller logs, while a trigger posts an activation requests, the rest of the request handling is buggy:

```
[2017-08-13T13:48:32.233Z] [WARN] [#tid_7] [TriggersApi] action guest/a could not be invoked due to Source(SourceShape(StatefulMapConcat.out), CompositeModule [3c60a25e] Name: iterableSource Modules: (singleSource) GraphStage(SingleSource(List(ByteString(123, 34, 97, 99, 116, 105, 118, 97, 116, 105, 111, 110, 73, 100, 34, 58, 34, 98, 97, 53, 99, 57, 51, 55, 102, 51, 97, 57, 101, 52, 48, 56, 52, 56, 98, 49, 102, 100, 54, 99, 53, 54, 54, 52, 55, 48, 100, 50, 50, 34, 125)))) [41c5bf50] (unnamed) [703a0c97] copy of GraphStage(StatefulMapConcat) [50915aa5] Downstreams: single.out -> StatefulMapConcat.in Upstreams: StatefulMapConcat.in -> single.out MatValue: Atomic(singleSource[41c5bf50])) 
```

### PR DESCRIPTION
**Pull request by rabbah**
Fixes #2611.

### CODE REVIEW

File: core/controller/src/main/scala/whisk/core/controller/Triggers.scala [hunk 1]
```diff
@@ -173,20 +171,25 @@ trait WhiskTriggersApi extends WhiskCollectionAPI {
...
-                                            logging.info(this, s"action ${rule.action} could not be found")
-                                        case HttpResponse(code, _, entity, _) =>
-                                            logging.warn(this, s"action ${rule.action} could not be invoked due to ${entity.getDataBytes.toString}")
+                                    Http().singleRequest(request).map { response =>
+                                        response.status match {
+                                            case OK | Accepted => Unmarshal(response.entity).to[JsObject].map { a =>
+                                                logging.info(this, s"${rule.action} activated ${a.fields("activationId")}")
+                                            }
+                                            case NotFound =>
+                                                logging.info(this, s"${rule.action} failed, action not found")
```
Comment (markusthoemmes) [inline core/controller/src/main/scala/whisk/core/controller/Triggers.scala]: You need to consume the bytes in any case. If you're not using them use Sink.ignore

Comment (rabbah) [inline core/controller/src/main/scala/whisk/core/controller/Triggers.scala]: Or response.discardEntityBytes()?

Thx for the catch.

Comment (markusthoemmes) [inline core/controller/src/main/scala/whisk/core/controller/Triggers.scala]: I'm on mobile, can't check the implementation but sounds like the helper you're looking for, yes.

**Comment by rabbah** [comment]
commit amended @markusthoemmes.

**Comment by rabbah** [comment]
Pg3/1004 🔵

**Comment by rabbah** [comment]
Should probably shutdown the http connections on exit.

**Pull request merged**